Lib/asyncio/exceptions.py

```diff
@@ -10,8 +10,7 @@ class CancelledError(BaseException):
     """The Future or Task was cancelled."""
 
 
-class TimeoutError(Exception):
-    """The operation exceeded the given deadline."""
+TimeoutError = TimeoutError  # make local alias for the standard exception
```

Comment:
Would it be possible to move this one to `asyncio/__init__.py`?

Please mention "bpo-42413:" in the comment. Like:

```
# bpo-42413: asyncio.TimeoutError is now an alias to the builtin TimeoutError exception.
TimeoutError = TimeoutError
```

Reply:
We can but it increases a chance of backward incompatibility.
People still can use `from asyncio.exceptions import TimeoutError`.
I dislike this style and strive to avoid it personally, but I recall a lot of third-party code that uses such imports.

The main reason is that IDE or another tool can insert `obj.__class__.__module__` when applies 'autoimport' to the edited python file.
PyCharm, for example, changed this behavior to substitute the top-most name (perform a lookup in `__init__.py` for the package) about a year ago after I discussed the feature with PyCharm devs at US PyCon 2019.
I'm not sure what other tools do.
Ideally, we can add some trick with module-level `__getattr__` to raise a warning about bad usage but I'm really not sure if we should do it this PR.